core/domain/feedback_services_test.py

```diff
@@ -232,7 +232,52 @@ def test_get_all_threads_without_suggestion(self):
             with self.swap(utils, 'get_time_in_millisecs',
                            self._get_milliseconds):
                 threads = feedback_services.get_all_threads(
-                    self.EXP_ID2, False)
+                    self.EXP_ID1, False)
```

Comment:
Oh, I see what you mean. Yep, that is bad, and I agree with what you've done.

In fact, could we go ahead with removing the swap altogether? There's a tradeoff here between testing too much stuff and dealing with hidden dependencies behind the scenes, and I'm more concerned about the latter than the former.